.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,32 @@
+name: Report a bug
+description: Something is not working as expected? Tell us the problem
+labels: ["kind/bug"]
```

Comment:
We should have additional label here in my opinion. Saying it's a new to be triaged. ["needs-triage"], because it should be one from:
  https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.md

Reply:
alright, going to add